public/assets/js/index.js

```diff
@@ -1 +1,13 @@
-/* JS code to be rendered to the web browser */
\ No newline at end of file
+var Btn = document.getElementById('Btn');
```

Comment:
The only thing I would say for this is to make the var name more descriptive. What does this button do? Just adding this incase we need to change this button or something in the future, we should know what it does

Reply:
finished